packages/terra-heading/src/terra-dev-site/doc/example/HeadingColors.jsx

```diff
@@ -3,6 +3,31 @@ import Heading from 'terra-heading';
 import classNames from 'classnames/bind';
 import styles from './colors.module.scss';
 
+// By passing in a className, authors can manually set the text color of any `<Heading />` component.
+
+// :root {
+//   --my-app-heading-attention-color: rgb(220, 20, 60);
+//   --my-app-heading-success-color: rgb(46, 125, 50);
+//   --my-app-heading-info-color: #304ffe;
+//   --my-app-heading-default-color: #333;
+// }
+
+// .default {
+//   color: var(--my-app-heading-default-color, #333);
+// }
+
+// .attention {
+//   color: var(--my-app-heading-attention-color, rgb(220, 20, 60));
+// }
+
+// .success {
+//   color: var(--my-app-heading-success-color, rgb(46, 125, 50));
+// }
+
+// .info {
+//   color: var(--my-app-heading-info-color, #304ffe);
+// }
```

Comment:
If these comments are here to demonstrate theming/custom colors I think that information would be better suited to be presented in the upgrade guide or in a more coherent document (perhaps a high-level theming strategy document). Commented out code in the JSX file is a little confusing.

We discussed adding a CSS tab to the code examples a while back, but we decided to hold off on that functionality. In this case it may be beneficial.

Reply:
I had the same question posted here https://github.com/cerner/terra-core/pull/2863#issuecomment-591803668. 
What I believe is we can leave `CSS` in the example itself until the toggle CSS functionality is introduced.